integration-tests/src/lib.rs

```diff
@@ -65,7 +65,7 @@ impl Default for NodeConfig {
                 account_sk: "fS5jS6X5qvaquBV1bg2YWBdYeCiRSUwNAdNpgNkjS72oNxUJcZJZduaq2oCcXJb8erTbtqqq4wxriUmRJk7bMDw"
                     .to_string(),
                 rpc_url: "https://api.devnet.solana.com".to_string(),
-                program_address: "BtGZEs9ZJX3hAQuY5er8iyWrGsrPRZYupEtVSS129XKo".to_string(),
+                program_address: "CMGYAEsqXw5z52R8fmMZwPYQARHPEkGbefJA2FmeHLMh".to_string(),
```

Comment:
We talked with Osman and agreed to keep that address as our dev env program. Is it necessary to change it?

Reply:
I can revert this one. It is for local testing. For my local testing, the BtG... address is not working with Osman's script, maybe I need to start from fresh, rebuild the contract from latest commit.

Reply:
Thhat contract was not set as immutable and controlled by your key, you can update it.